finetune/base.py

```diff
@@ -308,7 +320,11 @@ def _featurize(self, *Xs, max_length=None):
             warnings.filterwarnings("ignore")
             max_length = max_length or self.config.max_length
             for xmb, mmb in self._infer_prep(*Xs, max_length=max_length):
-                feature_batch = self.sess.run(self.features, {self.X: xmb, self.M: mmb})
+                feature_batch = self.sess.run(self.features, {
```

Comment:
Interesting, was this broken or has something changed with the logic to add dropout ops?